internals/secrethub/write_test.go

```diff
@@ -28,6 +29,7 @@ func TestWriteCommand_Run(t *testing.T) {
 		promptOut   string
 		promptErr   error
 		passwordIn  string
+		clientErr   error
```

Comment:
I suggest naming this error `newClientErr` because that's the name used in other tests.